lib/resourcemerge/apps_test.go

```diff
@@ -0,0 +1,78 @@
+package resourcemerge
+
+import (
+	"testing"
+
+	appsv1 "k8s.io/api/apps/v1"
+	"k8s.io/apimachinery/pkg/api/equality"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/utils/pointer"
+)
+
+func TestEnsureDeployment(t *testing.T) {
+	replicas := int32(2)
+	expectedReplicas := int32(2)
+	labelSelector := metav1.LabelSelector{}
+	tests := []struct {
+		name     string
+		existing appsv1.Deployment
+		required appsv1.Deployment
+
+		expectedModified bool
+		expected         appsv1.Deployment
+	}{{
```

Comment:
this is different in format.. than others

Reply:
Can you please explain a bit what you mean by "other" format? I tried to follow the format of the tests recently got added in[ #99](https://github.com/openshift/cluster-version-operator/pull/199/files#diff-415c13f11ffc32696c5d69b900b3fe58R11). Also i checked couple more existing unit test files like status_test.go and sync_test.go, those too seem to have different formats.
I think i did not understand correctly what you refering to as "others".